src/System.Drawing.Primitives/src/System.Drawing.Primitives.csproj

```diff
@@ -1,34 +1,43 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
You shouldn't need to set netstandard1.7 TargetGroup here. we should just assume TargetGroup empty means netstandard1.7